pkg/sql/vars.go

```diff
@@ -1382,6 +1382,24 @@ var varGen = map[string]sessionVar{
 			return experimentalComputedColumnRewrites.Get(sv)
 		},
 	},
+
+	`enable_copying_partitioning_when_deinterleaving_table`: {
+		GetStringVal: makePostgresBoolGetStringValFn(`enable_experimental_stream_replication`),
```

Comment:
nit: use constant?

Reply:
The entire rest of the file uses the string twice 🤷